samcli/commands/_utils/template.py

```diff
@@ -29,20 +29,26 @@ class TemplateFailedParsingException(UserException):
     pass
 
 
-def get_template_data(template_file):
+def get_template_data(template_file, root_template_path: str = ""):
```

Comment:
do we expect `root_template_path` to be `None` or `""`?

Reply:
I think either is fine. Just need to keep it consistent in other places. My understanding is if it is string, we prefer using `""`.

Reply:
Updated it to None since it makes more sense.